enhancements/20231113-witness-node-support.md

```diff
@@ -0,0 +1,71 @@
+# Support witness node
+
+## Summary
+
+The witness node is a lightweight node that only runs the control plane components. The witness node is not schedulable and does not run any workload. The witness node forms a quorum with the other two nodes.
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/3266
+
+## Motivation
+
+### Goals
+
+Create a three-node cluster with one witness node.
+
+## Proposal
+
+### User Stories
+
+For the edge use case, the witness node support benefits both cost and high availability. The classical scenario would be two nodes and one witness node.
+
+### User Experience In Detail
+
+1. Set the node label `node-role.harvesterhci.io/etcd=true` to the node.
```

Comment:
Since it's user experience, suggest describing the installer change. Either interactive or iPXE way.

Reply:
OK, I will update once we have a conclusion of the above discussion.

Reply:
This needs to be handled by the installer. Could be as simple as adding a new panel which in turn populates the additional node labels.